packages/neuron-ui/src/styles/theme.scss

```diff
@@ -1,73 +1,79 @@
   --activity-color: #{$main-color};
   --success-color: #{$main-color};
-  --error-color: #FF1E1E;
-  --input-disabled-color: #FAFAFA;
+  --error-color: #ff1e1e;
+  --input-disabled-color: #fafafa;
   --input-second-color: #999999;
   --input-button-color: #999999;
   --button-cancel-color: #999999;
   --svg-color: #666666;
   --dialog-secondary-text-color: #666666;
-  --disable-button-text-color: #CCCCCC;
+  --disable-button-text-color: #cccccc;
   --enable-button-text-color: #333333;
   --main-pagination-button-text-color: #666666;
   --main-pagination-text-color: #666666;
-  --warn-background-color: #FFF6EB;
-  --svg-fill-color: #FFFFFF;
+  --warn-background-color: #fff6eb;
+  --svg-fill-color: #ffffff;
   // table
-  --table-head-background-color: #FAFAFA;
-  --table-head-border-color: #F0F0F0;
+  --table-head-background-color: #fafafa;
+  --table-head-border-color: #f0f0f0;
 
   @media (prefers-color-scheme: dark) {
     --primary-color: #{$main-color};
     --primary-text-color: #232323;
     --main-background-color: #111614;
-    --secondary-background-color: #1E2423;
-    --main-text-color: #FFFFFF;
-    --secondary-text-color: #8DA394;
+    --secondary-background-color: #1e2423;
+    --main-text-color: #ffffff;
+    --main-shadow-color: #141b1a;
+    --secondary-text-color: #8da394;
+    --third-text-color: #cccccc;
     --link-color: #{$main-color};
-    --third-background-color: #2D3534;
-    --notice-background-color: #093F29;
-    --notice-text-color: #F78C2A;
-    --border-color: #343E3C;
-    --input-border-color: #343E3C;
-    --input-hint-color: #FFFFFF;
+    --third-background-color: #2d3534;
+    --fourth-background-color: var(--secondary-background-color);
+    --notice-background-color: #093f29;
+    --notice-text-color: #f78c2a;
+    --border-color: #343e3c;
+    --input-border-color: #343e3c;
+    --input-hint-color: #ffffff;
     --divide-line-color: rgba(240, 240, 240, 0.2);
     --step-no-activity-color: #666666;
     --activity-color: #{$main-color};
     --success-color: #{$main-color};
-    --error-color: #FF1E1E;
-    --input-disabled-color: #171B1A;
+    --error-color: #ff1e1e;
+    --input-disabled-color: #171b1a;
     --input-second-color: #999999;
-    --input-button-color: #FFFFFF;
+    --input-button-color: #ffffff;
     --button-cancel-color: #999999;
-    --svg-color: #FFFFFF;
+    --svg-color: #ffffff;
     --dialog-secondary-text-color: #999999;
     --disable-button-text-color: #666666;
-    --enable-button-text-color: #FFFFFF;
-    --main-pagination-button-text-color: #CCCCCC;
+    --enable-button-text-color: #ffffff;
+    --main-pagination-button-text-color: #cccccc;
     --main-pagination-text-color: #999999;
     --warn-background-color: rgba(255, 140, 0, 0.2);
     --svg-fill-color: #333333;
     // table
-    --table-head-background-color: #171B1A;
-    --table-head-border-color: #282E2D;
+    --table-head-background-color: #171b1a;
+    --table-head-border-color: #282e2d;
```

Comment:
We don't need to format color

Reply:
This is because the current automatic code style configuration is set up that way, and we shouldn't manually adjust the code style. We should create a separate issue later to address the configuration problem with the automation of the code style.